Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -8,17 +8,38 @@
 
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Instances page](#instances-page)
-* [Deploying a New Instance](#deploying-a-new-instance)
-* [Scheduling Instances](#scheduling-instances)
-* [Protecting Instance Shutdown or Termination](#protecting-instance-shutdown-or-termination)
-* [Deploying Instances without Lifecycle Management](#deploying-instances-without-lifecycle-management)
-* [Handling Instance Lifecycle States](#handling-instance-lifecycle-states)
-* [Managing multiple instances with bulk actions](#managing-multiple-instances-with-bulk-actions)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#overview)
```

Comment:
Please, leave * as the bullet points symbol, and only show 3 level titles in the ToC